tyr-core/src/main/java/org/jboss/tyr/model/Utils.java

```diff
@@ -50,6 +50,10 @@
     public static final String MESSAGE = "message";
     public static final String URL = "url";
 
+    //Repository verification regex
+    public static final String REPOSITORY_FORMAT_VERIFICATION_REGEX =
+            "^([a-zA-Z0-9](?:-?[a-zA-Z0-9]){0,38})/[a-zA-Z0-9_.-]{1,100}$";
```

Comment:
just wondering if you found this limitations somewhere? Are these official rules?

Reply:
Numbers of characters are official, username format also official. Googled repository name format and tested it. (you can also choose other characters in repository name but they get replaced by `-`)

Reply:
> 
> 
> just wondering if you found this limitations somewhere? Are these official rules?

Max characters are official, username format also official. Googled repository name format and tested it. (you can also choose other characters in repository name but they get replaced by `-`)